src/main/resources/templates/admin.html

```diff
@@ -49,26 +45,21 @@ <h1>Pending Sightings</h1>
                                     <span th:text="${sighting.getId()}"></span>
                                 </div>
                             </div>
-
                             <div class="checkboxgroup">
                                 <label for="radio-button-id1">Reject</label>
-                                <input type="radio" th:name="${sighting.getId()}" value="reject" id="radio-button-id1" />
+                                <input type="radio" th:name="${sighting.getId()}" value="0" id="radio-button-id1" />
```

Comment:
can we use an enum or boolean here instead of 0 and 1?

Reply:
I will investigate to see how to do boolean or enum into thymeleaf